packages/keystone/crypto/EncryptionManager.js

```diff
@@ -0,0 +1,227 @@
+const crypto = require('crypto')
+
+const get = require('lodash/get')
+const isEmpty = require('lodash/isEmpty')
+const isNil = require('lodash/isNil')
+
+const conf = require('@open-condo/config')
+
+
+// since data is converted in hex, ':' shouldn't be in it
+const SEPARATOR = ':'
+const SUPPORTED_MODES = ['cbc', 'ctr', 'cfb', 'ofb', 'ecb']
+const SUGGESTIONS = {
+    'ecb': 'WARNING! Don\'t use "ecb" in real application as it is insecure!',
+    'cfb': 'Please, consider using "ctr" or "cbc"',
+    'ofb': 'Please, consider using "ctr" or "cbc"',
+}
+
+/**
+ * @typedef {Object} EncryptionManagerOption
+ * @property {string} id - version unique key
+ * @property {string} algorithm - crypto algorithm
+ * @property {string} secret - secret key
+ */
+
+/**
+ * @typedef EncryptionVersion
+ * @type {{id: string, algorithm: string, secret: string, ivLength: number}}
+ */
+
+
+/**
+ * Used for versioning secrets for encryption / decryption
+ * @example
+ * const manager = new EncryptManager() // use versions from .env DEFAULT_KEYSTONE_ENCRYPTION_CONFIG
+ * const manager = new EncryptManager({ customConfig: [{ id: '22', algorithm: 'aes-256-cbc', secret: <...> }] }) // encode with you version, decode from your and default versions
```

Comment:
Why double-decode?

Reply:
I'm not sure if this API is friendly, so many questions...

Why it's called a custom, not just a config
Why it's an Array? Not a single algo...

Why default is included implicitly, it can cause bug. 

Why not something like:
```
const manager = new EncryptManager(methods: [{...}, {...}, EncryptManager.defaultMethod()]) // via static method or const 
```